src/Core/MailTemplates/Handlebars/UpdatedTempPassword.text.hbs

```diff
@@ -0,0 +1,3 @@
+{{#>BasicTextLayout}}
+The temporary master password set by an administrator for {{UserName}} has been changed. If you did not initiate this request, please reach out to your administrator immediately.
```

Comment:
Does this content/text fall in-line with expectations or all use-cases when this email would be sent? cc: @ginat

Reply:
Imo, as of right now - yes. This email will only be sent when calling this API for updating a temp password set by an admin. I agree though - when everyone gets on it, we can easily change the verbiage 👍